apps/nextjs-website/react-components/types/Hero/Hero.types.ts

```diff
@@ -14,10 +14,16 @@ export interface StoreButtonsProps {
   readonly hrefApple?: string;
 }
 
+export interface ChipProps {
+  readonly label: string;
+  readonly targetId: string;
+}
+
 export interface HeroTextProps extends CommonProps {
   readonly title: string;
   readonly subtitle?: string | Generic;
   readonly ctaButtons?: ReadonlyArray<CtaButtonProps>;  
   readonly storeButtons?: StoreButtonsProps;
   readonly size?: 'medium' | 'big' | 'small';
+  readonly chips?: ReadonlyArray<ChipProps>;
```

Comment:
```suggestion
  readonly chips: ReadonlyArray<ChipProps>;
```
It's probably better to make `chips` required and check its length to determine whether or not to render any chips.